src/examples/accordion-example.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import Example from './example.js';
+import { Accordion, Section } from '../accordion';
+
+function CustomTitle({ open, title }) { // eslint-disable-line react/prop-types
+  return (
+    <React.Fragment>
+      <span>{title}</span>
+      <span style={{ float: 'right' }}>{open ? '-' : '+'}</span>
+    </React.Fragment>
+  );
+}
+
+export default class AccordionExample extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      first: false,
+      second: false,
+      third: false,
+    };
+  }
+
+  handleClick(which) {
+    return () => {
+      this.setState(state => ({
+        first: false,
+        second: false,
+        third: false,
+        [which]: !state[which],
+      }));
+    };
+  }
+
+  render() {
+    return (
+      <Example title="Accordion">
+        <Accordion>
+          <Section
+            headingLevel={3}
+            onClick={this.handleClick('first')}
```

Comment:
Just realized this function returns new functions on every render. We could have created a `handleClickFirst = () => { this.handleClick('first') }` but it doesn't really matter here.

Reply:
Yeah, you're right. A better or real-world example would probably want to have separate handlers for each section, as you've suggested.

👍